cluster-up/OWNERS

```diff
@@ -0,0 +1,6 @@
+reviewers:
```

Comment:
`cluster-up` is a copy of the kubevirtci. I don't think we need a kubevirt owner for it.

Reply:
Even though the process is largely automated, there are still situations where we need to look into things. My intent was

a) to be clear about who is responsible for the bumping process and
b) to make sure that `sig-buildsystem` is assigned to the kubevirtci bumps, so that problems arising are picked up directly by us.

Doesn't that make sense?

Reply:
Yes, makes sense. Thanks for explaining.